src/c++/devices/dory/dmDory.c

```diff
@@ -1207,8 +1207,8 @@ static struct target_type doryTargetType = {
 /**********************************************************************/
 int __init doryInit(void)
 {
-  STATIC_ASSERT(offsetof(DoryDevice, dev) 
-                == offsetof(CommonDevice, dev)); 
+  BUILD_BUG_ON(offsetof(DoryDevice, dev)
+               != offsetof(CommonDevice, dev));
```

Comment:
If you're going to unwrap the line, I  think you should unwrap the line in all five test devices.